lib/windows/seekdialog.py

```diff
@@ -314,27 +316,64 @@ def doClose(self, delete=False):
         finally:
             kodigui.BaseDialog.doClose(self)
 
-    def determineSkipStep(self, direction, reset=False):
-        if reset:
-            self._atSkipStep = -1
-            self._lastSkipDirection = direction
-
+    def determineSkipStep(self, direction):
         stepCount = len(self.skipSteps[direction])
+
+        # shortcut for simple skipping
         if stepCount == 1:
             return self.skipSteps[direction][0]
```

Comment:
The boundary clamping should be checked here as well (`self._forcedLastSkipAmount`).
Not a big issue as we're talking a minimal boundary overlap at best, with +30 or +10.

Do you want me to add that?